### PR TITLE
change context to accounts and add roles table

### PR DESCRIPTION
**Pull request by GettyOrawo**
**Done**

Change context from User to Accounts to avoid redundancy since there's already a module called User.
Create a form with functionality for creating a role.
 - Roles should not be blank
 - Roles should be unique
Context tests for the above functionality

**To be done**

 - Every created user should belong to a role.
 - A role can have many users.

roles

### CODE REVIEW

File: lib/nailinda/accounts/accounts.ex [hunk 1]
```diff
@@ -73,4 +74,66 @@ defmodule Nailinda.Accounts do
...
+
+  def create_doctor(attrs) do
+    %Doctor{}
+    |> Doctor.changeset(attrs)
+    |> Repo.insert()
+  end
+
+  def get_doctor_by_id(id) do
+    Doctor
+    |> Repo.get!(id)
```
Comment (sigu) [inline lib/nailinda/accounts/accounts.ex:112]: if this repo function has a `!` then the main function should also contain it i.e `get_doctor_by_id!` to communicate to us that the function will throw an error if no result is found

**Review by sigu — APPROVED** [review]
Great one, opening an issue to resolve my comment. Should not be done on this PR

File: lib/nailinda/accounts/accounts.ex (hunk 1 above)
Comment (sigu) [reply lib/nailinda/accounts/accounts.ex:112]: resolved by https://github.com/Podiihq/nailinda/issues/103

**Comment by GettyOrawo** [comment]
This merge request has been opened in another branch. closing... #109

**Pull request closed without merging**